### ISSUE TITLE
Casing of Prisma models is changed in the type definitions, causing TS errors

### ISSUE THREAD

**Issue by douira** [chrishoermann/zod-prisma-types#38]
I have an enum called `UNMemberType` in my Prisma schema and the corresponding type generated by Prisma is also called that. However, when zod-prisma-types generates the zod models, it tries to access `PrismaClient.UnMemberType` which doesn't exist. Only `PrismaClient.UNMemberType` exists.

Maybe the names of the types could simply be used as they are without modifying their casing?

**Comment by douira** [comment]
Additionally I've noticed that while the generated zod model is called `UnMemberTypeSchema`, when it later accesses this model in the file, it tries to use `UNMemberTypeSchema` which is also wrong. These issues may be related but they aren't quite the same. Example:

In `schema.prisma`:
```prisma
enum UNMemberType {
  ...
}
```

Generated Prisma types in the prisma client:
```ts
export const UNMemberType: {
  ...
};
```

Generated zod models in `index.ts`:
```ts
export const UnMemberTypeSchema = z.nativeEnum(PrismaClient.UnMemberType);

...

export const DelegationSchema = z.object({
  memberType: UNMemberTypeSchema,
  id: z.string(),
  shortName: z.string(),
  fullName: z.string(),
});
```

**Comment by douira** [comment]
Is there a specific reason it converts the names into various forms of camel case throughout the generation process? Are the default names provided by the schema insufficient? If this is just a stylistic choice, it would be good if this could be disabled.

**Comment by chrishoermann** [comment]
Thanks for the info @douira! You where right that both issues where related because I used my `formattedNames`, that I create in each class for different use cases, instead of the actual model name. In version 1.3.1 the following model

```prisma
enum MYValue {
  A
  B
  C
}
```

generates this enum schema

```ts
export const MYValueSchema = z.nativeEnum(PrismaClient.MYValue);
```

that is based on the prisma generated enum

```ts
export const MYValue: {
  A: 'A',
  B: 'B',
  C: 'C'
};
```

**Comment by douira** [comment]
Thanks for the fix. Does this also apply to the casing of other aspects of a schema like models and their contents? For example, what if I can come up with the following example of various formatting that may exist for existing (and potentially unchangeable) code:
```prisma
model MembersOfTheUN {
  SIZE             Int          @id
  UNMemberType     UNMemberType
  internal_counter Int
}
```

**Comment by chrishoermann** [comment]
I checked and indeed I used the wrong casing in some cases - so model and field names with uppercase would have caused problems.
I fixed it in version 1.3.3.

So the following model 
```prisma
enum MYValue {
  A
  B
  C
}

model MODELWithUpperCase {
  id   Int  @id @default(autoincrement())
  STRING String
  MYValue MYValue
}
```

will generate the following output now:

```ts
export const MYValueSchema = z.nativeEnum(PrismaClient.MYValue);

export const MODELWithUpperCaseScalarFieldEnumSchema = z.nativeEnum(PrismaClient.Prisma.MODELWithUpperCaseScalarFieldEnum);

export const MODELWithUpperCaseSchema = z.object({
  MYValue: MYValueSchema,
  id: z.number(),
  STRING: z.string(),
});

export const MODELWithUpperCaseSelectSchema: z.ZodType<PrismaClient.Prisma.MODELWithUpperCaseSelect> = z.object({
  id: z.boolean().optional(),
  STRING: z.boolean().optional(),
  MYValue: z.boolean().optional(),
}).strict();


export const MODELWithUpperCaseCreateInputSchema: z.ZodType<PrismaClient.Prisma.MODELWithUpperCaseCreateInput> = z.object({
  STRING: z.string(),
  MYValue: z.lazy(() => MYValueSchema),
}).strict();

export const MODELWithUpperCaseUncheckedCreateInputSchema: z.ZodType<PrismaClient.Prisma.MODELWithUpperCaseUncheckedCreateInput> = z.object({
  id: z.number().optional(),
  STRING: z.string(),
  MYValue: z.lazy(() => MYValueSchema),
}).strict();

// and so on


```
